package.json

```diff
@@ -122,7 +122,7 @@
     "Markus Siebert <mail@markussiebert.com>",
     "Sergey Vedmak <serg.vedmak@gmail.com>"
   ],
-  "license": "AGPL-3.0",
+  "license": "AGPL-3.0-only",
```

Comment:
I'll let you review this section of the license:

https://github.com/renovatebot/renovate/blob/1c898a1cf185b921ded6d05c1bf7192caaf2bd9b/license#L569-L593